py_mmd_tools/nc_to_mmd.py

```diff
@@ -992,16 +992,23 @@ def get_related_information(self, mmd_element, ncin):
                     '%s must contain valid uris' % acdd_key
                 )
                 continue
-            type = ri[1][:-1]
-            if type not in VALID_TYPES:
+            ref_type = ri[1][:-1]
+            valid_ref_types = [vt.lower() for vt in VALID_REF_TYPES]
+            if ref_type.lower() not in valid_ref_types:
                 self.missing_attributes['errors'].append(
-                    'Reference types must follow a controlled '
-                    'vocabulary from MMD (see https://htmlpreview.'
-                    'github.io/?https://github.com/metno/mmd/blob/'
-                    'master/doc/mmd-specification.html#related-'
-                    'information-types).')
+                    'Reference types must follow a '
+                    'controlled vocabulary from MMD (see '
+                    'https://htmlpreview.github.io/?https://github.'
```

Comment:
https://vocab.met.no/mmd/Related_Information_Types is probably a better link

Reply:
hmm - the source is anyway the same but I agree. The problem is that then we should change all the other links to MMD controlled vocabularies as well. Maybe we could do that in another issue?